README.markdown

```diff
@@ -95,6 +95,65 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+**NOTE**: There is a bug currently, that causes FakeFS to fail with the following error:
+```
+TypeError:
+  superclass mismatch for class File
+```
+
+To prevent this, you'll need to require 'pp' *BEFORE* you require 'fakefs/spec_helpers' in your
+spec_helpers file. Like this:
+
+```ruby
+require 'pp'
+require 'fakefs/spec_helpers'
+
+describe "my spec" do
+  include FakeFS::SpecHelpers
+end
+```
+
+Mocking relative path in a Gem
+------------------------------
+
+If you are working in a Gem and would like to load configuration files which sit with your gem in a
+cfg folder, you will need to clone your filesystem first. To do this, add the following line to your
+spec_helpers file.
+
+```ruby
+...
+shared_examples 'with faked out filesystem' do
+  before :each do
+    include FakeFS::SpecHelpers
+    FakeFS.activate!
+    FakeFS::FileSystem.clone(__dir__)
+  end
+end
+...
+```
+
+And than if you have a test which needs to use a faked filesystem, like a configuration test, you'll
```

Comment:
... don't think this needs any extra comment, it's pretty obvious that it is setup above and then getting used here ... I'd even get rid of the shared example stuff to make it simpler ...